stan/math/rev/core/typedefs.hpp

```diff
@@ -5,6 +5,11 @@
 #include <stan/math/rev/meta.hpp>
 #include <stan/math/rev/core/var.hpp>
 #include <stan/math/rev/core/Eigen_NumTraits.hpp>
+#include <chrono>
+#include <string>
+#include <thread>
+#include <map>
+#include <utility>
```

Comment:
I think put the ones you need in the profiling header, unless there's a reason to put them here.